Automattic/wp-calypso
# Activity Log: Introduce event sharing

#### Changes proposed in this Pull Request

* Introduces event sharing on the activity log in Calypso Green.
* Add state and data layer support for POSTing to `/activity-log/{siteId}/share/{activityId}`.
* Adds rough css to style the sharing button and popover.
* Adds success and error notices to indicate whether sharing has succeeded or failed.

#### Testing instructions

* Visit the activity log in Calypso Green dev. You should see "Share this event" at the bottom of most events.
* Clicking the share button should display a popover with an email address field and submit button.
* Email addresses should loosely validate.
* Submitting the form should result in an error notice (the API is not yet shipped).



Related to #
